applications/endoscopy_out_of_body_detection/README.md

```diff
@@ -1,68 +1,137 @@
+## Prerequisites
+
+- NVIDIA Holoscan SDK (version 0.5 or higher)
+- [A supported Holoscan platform or workstation with CUDA-capable NVIDIA GPU](https://docs.nvidia.com/holoscan/sdk-user-guide/sdk_installation.html)
+- CMake build system
+- FFmpeg (for data conversion)
+
+## Data Requirements
+
+### Model and Sample Data
+
+The endoscopy detection model and sample datasets are available on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/endoscopy_out_of_body_detection). The package includes:
 
-# convert video file into tensor
-ffmpeg -i <INPUT_VIDEO_FILE> -fs 900M -pix_fmt rgb24 -f rawvideo pipe:1 | python convert_video_to_gxf_entities.py --width 256 --height 256 --channels 3 --framerate 30
+- Pre-trained ONNX model for out-of-body detection: `out_of_body_detection.onnx`
+- Sample endoscopy video clips (MP4 format): `sample_clip_out_of_body_detection.mp4`
 
-# where <INPUT_VIDEO_FILE> is one of the downloaded MP4 files: OP1-out-2.mp4, OP4-out-8.mp4 or OP8-out-4.mp4.
+### Data Preparation (optional)
+
+The application requires the input videos to be converted to GXF tensor format. This conversion happens automatically during building, but manual conversion can be done following these steps:
+
+1. Download and extract the data:
+
+```bash
+unzip [NGC_DOWNLOAD].zip -d <data_dir>
 ```
 
-Move the model file and converted video tensor into a directory structure similar to the following:
+2. Convert video to GXF tensor format using the provided script:
 
 ```bash
-data
-└── endoscopy_out_of_body_detection
+ffmpeg -i <INPUT_VIDEO_FILE> -fs 900M -pix_fmt rgb24 -f rawvideo pipe:1 | \
+python convert_video_to_gxf_entities.py --width 256 --height 256 --channels 3 --framerate 30
+```
+
+Note: The conversion script (`convert_video_to_gxf_entities.py`) is available in the [Holoscan SDK repository](https://github.com/nvidia-holoscan/holoscan-sdk/tree/main/scripts).
+
+3. Organize the data directory as follows:
+
+```
+data/
+└── endoscopy_out_of_body_detection/
     ├── LICENSE.md
     ├── out_of_body_detection.onnx
     ├── sample_clip_out_of_body_detection.gxf_entities
     ├── sample_clip_out_of_body_detection.gxf_index
     └── sample_clip_out_of_body_detection.mp4
 ```
 
-## Building the application
+## Configuration
+
+The application uses `endoscopy_out_of_body_detection.yaml` for configuration. Key settings include:
+
+- Input video parameters in the `replayer` section
+- Model parameters in the `inference` section
+- Analytics settings for data export
+
 
-Please refer to the top level Holohub README.md file for information on how to build this application.
+## Building
 
-Additionally, if the Holoscan SDK version is 0.5 or lower, following code changes must be made in the application:
+./dev_container build  # do we need a custom Dockerfile for ffmpeg?
+./dev_container launch 
+./run build endoscopy_out_of_body_detection <cpp/python>
+./run launch endoscopy_out_of_body_detection <cpp/python>
```

Comment:
Confirmed ffmpeg is the default HoloHub container so we do not need a custom Dockerfile here. Might need to clean up formatting with a markdown code block

```suggestion
./dev_container build
./dev_container launch 
./run build endoscopy_out_of_body_detection <cpp/python>
./run launch endoscopy_out_of_body_detection <cpp/python>
```